.release-please-manifest.json

```diff
@@ -1,3 +1,3 @@
 {
-  ".": "0.6.3"
+  ".": "0.7.0"
```

Comment:
Confirmed manifests releaser have worked properly, and be aware that for using `release-please-config.json`, we have to prepare `.release-please-manifest.json` with this format.